app/src/shared/hooks/useOnceEffect.js

```diff
@@ -0,0 +1,9 @@
+import { useRef, useEffect } from 'react'
+
+export default (callback) => {
```

Comment:
IMO this should be a named function or variable otherwise it'll be anonymous in any stack traces

https://twitter.com/dan_abramov/status/1280848859275628545